### PR TITLE
fix: txid not found when clicking the transaction toast immediately

### PR DESCRIPTION
**Pull request by saujanyanagpal104**
## Description

Added 5 seconds setTimeout on toast to fix the txid  not found issue.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

After completing a successful send of a BTC transaction, a green transaction toast window will appear in the top right of your screen. Clicking the link immediately after the BTC send in the transaction toast will take you to an invalid explorer page because the TXID is not found yet.

## Risk

No risks, non-breaking.

## Testing

1. Make a BTC send transaction.
2. Observe the toast window pop up in the top right of the screen.
3. Immediately click on the link within the toast window to view transaction on an explorer page.

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Hi @saujanyanagpal104, thanks for the contribution! Could you please rename your PR to fit the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standards? 🙏🏽

**Comment by gomesalexandre** [comment]
Also, make sure all the fields of the description are filled. They are prefilled when creating a PR and you shouldn't remove them.

**Comment by saujanyanagpal104** [comment]
> Hi @saujanyanagpal104, thanks for the contribution! Could you please rename your PR to fit the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standards? 🙏🏽

Updated!

**Comment by saujanyanagpal104** [comment]
> Also, make sure all the fields of the description are filled. They are prefilled when creating a PR and you shouldn't remove them.

Updated!

**Comment by gomesalexandre** [comment]
> > Hi @saujanyanagpal104, thanks for the contribution! Could you please rename your PR to fit the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standards? 🙏🏽
> 
> Updated!

Still missing issue #, risks, and testing steps.

**Comment by saujanyanagpal104** [comment]
> 

Updated.

**Review by gomesalexandre — APPROVED** [review]
OK, toast is delayed ~5 seconds and block explorer link is now working when clicking the link in toast.

**Comment by saujanyanagpal104** [comment]
@gomesalexandre Hey, I think a check is failed due to commit message. What can we do to fix that now?

**Comment by gomesalexandre** [comment]
> @gomesalexandre Hey, I think a check is failed due to commit message. What can we do to fix that now?

You're missing the type in your commit, amend it to add it: `git commit --amend --no-edit -m "fix: added settimeout on toast"`
Don't forget to push an empty commit with `git commit --allow-empty -m "chore: trigger CI"` as force-pushing your amended commit won't properly CI.

**Comment by 0xdef1cafe** [comment]
@saujanyanagpal104 tests are failing

**Comment by 0xdef1cafe** [comment]
@saujanyanagpal104 you can run `yarn local-ci` rather than pushing commits to test that things will pass too, you might find that more helpful.

**Comment by saujanyanagpal104** [comment]
> 

Just checked build failed this is something completely unrelated to the changes I made! Let me check!

**Comment by saujanyanagpal104** [comment]
> @saujanyanagpal104 you can run `yarn local-ci` rather than pushing commits to test that things will pass too, you might find that more helpful.

All the tests passed with local-ci. Changes pushed. Workflows need approval I think. @gomesalexandre

File: src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx [hunk 1]
```diff
@@ -159,8 +159,10 @@ describe.each([
 
     const { result } = renderHook(() => useFormSend())
     await result.current.handleSend(formData)
-    expect(toaster).toHaveBeenCalledWith(expect.objectContaining({ status: 'success' }))
-    expect(sendClose).toHaveBeenCalled()
+    setTimeout(() => {
```
Comment (0xdef1cafe) [inline src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx:162]: `setTimeout` in tests is not the correct approach, please mock and advance timers.

File: src/features/defi/components/Deposit/Deposit.tsx [hunk 2]
```diff
@@ -313,7 +313,7 @@ export const Deposit = ({
                             inputMode='decimal'
                             thousandSeparator={localeParts.group}
                             value={value}
-                            onChange={e => {
+                            onChange={(e: any) => {
```
Comment (0xdef1cafe) [inline src/features/defi/components/Deposit/Deposit.tsx:316]: unrelated please revert all of these

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

File: src/features/defi/components/Deposit/Deposit.tsx (hunk 2 above)
Comment (0xdef1cafe) [reply src/features/defi/components/Deposit/Deposit.tsx:316]: or more strongly type them

File: src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx (hunk 1 above)
Comment (saujanyanagpal104) [reply src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx]: Pushed these changes.

**Comment by saujanyanagpal104** [comment]
Workflow stuck. @gomesalexandre

**Review by 0xdef1cafe — APPROVED** [review]
@saujanyanagpal104 approved -  but in general a nit pick: if you're going to use a constant and repeat it somewhere abstract it out so when someone changes the delay in future the tests won't break

**Comment by saujanyanagpal104** [comment]
@gomesalexandre @0xdef1cafe Can I know when I get paid for this bounty I picked this from gitcoin and this was my first so I don't know about the process, just asking?

**Comment by 0xApotheosis** [comment]
Hey @saujanyanagpal104, @purelycrickets will get it sorted for you sometime this coming week 👍

**Comment by saujanyanagpal104** [comment]
> Hey @saujanyanagpal104, @purelycrickets will get it sorted for you sometime this coming week 👍

Thanks a lot! @0xApotheosis

**Pull request merged**